index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+        <header>
+            <nav>
+                <ul>
+                    <li>Home</li>
+                    <li>Studies</li>
+                    <li>Projects</li>
+                    <li>About</li>
+                </ul>
+            </nav>
+        </header>
+        <section>
+            <h1>Hey, i'm Julian</h1>
+            <p>I'm a Web and Software Developer based in Bogotá, Colombia. Specializing in building scalable, efficient software solutions and responsive websites. Passionate about collaborating with teams, in order to deliver high-quality, impactful products.</p>
+        </section>
+        <main>
+            <div>
+                <h2>Academic Background</h2>
+                <div>
+                    <h3>Computer Engineering - Universidad de la Sabana - 2024-2028 </h3>
+                    <p>I am currently studying, acquiring knowledge in various technologies such as Java and Python, learning the applied engineering sciences, and participating in academic research groups.</p>
+                </div>
+                <div>
+                    <h3>Software Development - Coderhouse - 2023-2024 </h3>
+                    <p>I gained experience developing projects with HTML, CSS, SASS, Bootstrap, Javascript, ReactJS, React Native, Redux, SQLIte, and MongoDB. Using VS Code, Android Studio y XCODE.</p>
+                </div>
+                <div>
+                    <h3>Bachillerato - Colegio Bilingüe Hispanoamericano Conde Ansúrez - 2010-2024 </h3>
+                    <p>I excelled in technological and mathematical subjects, providing a solid foundation in logic and problem-solving. I participated in a robotics competition, representing my school nationally.</p>
+                </div>
```

Comment:
Esto debería ser una lista, puesto que está enunciando ítems descendientes de un mismo padre: en este caso su historial académico

```suggestion
          <ul>
             <li>
                    <h3>Computer Engineering - Universidad de la Sabana - 2024-2028 </h3>
                    <p>I am currently studying, acquiring knowledge in various technologies such as Java and Python, learning the applied engineering sciences, and participating in academic research groups.</p>
             </li>
             
             <li>...</li>
          </ul>      
          
```